tests/test_parser.py

```diff
@@ -260,3 +260,13 @@ def test_pdf_save_path():
     # Clean up
     os.remove(result["pdf_path"])
     os.rmdir("tests/outputs/temp")
+
+
+@pytest.mark.asyncio
+def test_page_nums():
+    sample = "examples/inputs/sample_test_doc.pdf"
+    parser_type = "LLM_PARSE"
```

Comment:
Did we check using `STATIC_PARSE`?

Reply:
I didn't check with `STATIC_PARSE` because in `sample_test_doc.pdf`, most pages are different to one another because of images, and since `STATIC_PARSE` can't read images, it's a bit difficult to make sure the correct pages were extracted.

Let me try expanding the test with a different document, just to be sure.

Reply:
Added

Reply:
Thanks 🙏